lib/chat_api_web/controllers/registration_controller.ex

```diff
@@ -26,7 +26,7 @@ defmodule ChatApiWeb.RegistrationController do
             # # obvious that a user invitation expires after one use.
             # ChatApi.UserInvitations.expire_user_invitation(invite)
             conn
-            |> send_registration_event()
+            |> send_registration_event(params["company_name"])
```

Comment:
looks like these changes are already in master, can you rebase to clean up the diff?